Cython/Utility/AsyncGen.c

```diff
@@ -170,6 +156,20 @@ typedef struct {
 #define _PyAsyncGen_MAXFREELIST 80
 #endif
 
+static __pyx_CoroutineObject *__Pyx_AsyncGen_New(
```

Comment:
Why was this function moved down?

Reply:
Because the `Generator.proto` block happened to end up before the definition of the module state struct, so it is not possible to use `__Pyx_Globals()` that early in the file. Anyway it felt like a good idea to only have a prototype in the `xxx.proto` section and the implementation further down.